packages/headless/src/features/facets/facet-set/interfaces/request.ts

```diff
@@ -8,23 +8,22 @@ import {
   Expandable,
   AllowedValues,
   CustomSort,
-  SpecificSortCriteriaExplicitAlphanumeric,
+  SpecificSortCriteriaExplicit,
 } from '../../facet-api/request';
 
 export const facetSortCriteria: FacetSortCriterion[] = [
   'score',
   'alphanumeric',
-  'alphanumericDescending',
   'occurrences',
   'automatic',
 ];
```

Comment:
While looking at the facetSortCriteria const , I see that it is only used in headless-insight-facet-options.ts .Why did you remove the alphanumericDescending from this const but not from the FacetSortCriterion ?

Reply:
> Why did you remove the alphanumericDescending from this const but not from the FacetSortCriterion ?

Because I'm a goofball that likes to cause chaos from time-to-time.

Joking aside, thanks for pointing this out! It was a mistake (see previous comment about `alphanumericDescending`) and I'll fix it now 🙂